.github/scripts/determine_bump.py

```
@@ -0,0 +1,97 @@
+def sanitize_body(body: str) -> str:
+    """Redact sensitive patterns then truncate to 1000 chars."""
+    if not body:
+        return ""
+    body = _SENSITIVE.sub("[REDACTED]", body)
+    return body[:1000]
+
+prev_tag = sys.argv[1]
+since = os.environ["SINCE"]
+repo = os.environ["REPO"]
+
+result = subprocess.run(
```

Ruff (0.15.4)

[error] `subprocess` call: check for execution of untrusted input

(S603)